src/Dashboard/DashboardRestController.php

```diff
@@ -61,7 +61,7 @@ public function itemsOwnedByCurrentUser(Request $request)
         $pageUrlFactory = new PageUrlGenerator(
             $request->query,
             $this->urlGenerator,
-            'dashboard-items',
+            $request->get('_route'),
```

Comment:
`_route` is enkel bedoeld om te testen, gebruik hier het correcte pad waarop je bind in de silex-app: `user-offers`.